.prettierignore

```diff
@@ -1,8 +1,10 @@
 node_modules
 .loom
+.vscode-test
```

Comment:
These output folders should also be added to the eslintignore and stylelintignore too

Reply:
Added to the other ignores 👍